app.log

```diff
@@ -0,0 +1,2 @@
+{"level":"INFO","time":"2023-06-12T17:43:54.667Z","pid":5640,"hostname":"רבקה","msg":"my app is listening on http://localhost:3000"}
```

Comment:
this file should be in gitignore

Reply:
בנות ניסו לעשות אותו לitignore אבל זה לא אלך